app-route.html

```diff
@@ -151,6 +151,14 @@
           readOnly: true
         },
 
+        /**
+         * Auto activate route if path empty
+         */
+        "auto-activate": {
```

Comment:
Please change to camelCase.

Polymer should automatically do kebab case to camel case conversion. It should be fine to call this `autoActivate` as Polymer will translate `<element auto-activate></element>` to `element.autoActivate`.

Reply:
Sorry, I did not know about this possibility of polymer